drltrace_src/drltrace.cpp

```diff
@@ -254,13 +266,48 @@ lib_entry(void *wrapcxt, INOUT void **user_data)
     if (mod != NULL)
         modname = dr_module_preferred_name(mod);
 
+    /* Build the module & function string, then compare to the white/black
```

Comment:
Would it better to move this functionality in a separate function for blacklist/whitelist checking?

Reply:
lines 270-298